ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml

```diff
@@ -52,6 +52,16 @@ tests:
     test:
     - ref: openshift-qe-node-density
     workflow: rosa-aws-sts
+- as: ocp-qe-perfscale-ci-tests-rosa-ingress-perf
```

Comment:
@krishvoor Please update the test name here to "perfscale-rosa-multiaz-ingress-perf-3nodes". I will update remaining test names soon. I have been working mostly on 4.13 yaml file and haven't gotten to others yet, but we will stick to the test names that are noted in 4.13 rosa test suite yml fille.

Reply:
I missed this, pushing changes.

Reply:
Updated, ptal